src/advanced/node-management.md

```diff
@@ -679,20 +690,22 @@ A JSON object with the following fields:
 GET {explorer_base_path}/blocks?count={count}&skip_empty_blocks={skip}&latest={latest}
 ```
 
-Returns the explored range and the corresponding headers. The range specifies the
-smallest and largest heights traversed to collect at most `count` blocks.
+Returns the explored range and the corresponding headers. The range specifies
+the smallest and largest heights traversed to collect at most `count` blocks.
 
 #### Parameters
 
 - **count**: integer  
   The number of blocks to return. Should not be greater than
   [`MAX_BLOCKS_PER_REQUEST`][github_explorer]
```

Comment:
I discovered that this link is currently broken in the published documentation. I suggest updating it to a versioned link (not `master`, but `v0.10.x`). I think anyone on the core team can provide it.

Reply:
This file is still under correction. It will be updated in the next couple of days